service/controller/v10/version_bundle.go

```diff
@@ -62,7 +67,7 @@ func VersionBundle() versionbundle.Bundle {
 			},
 			{
 				Name:    "containerlinux",
-				Version: "1632.3.0",
+				Version: "1688.5.3",
 			},
 			{
 				Name:    "docker",
```

Comment:
Docker version is 17.12.1 - that is the most important bit :)

Reply:
Ay it is indeed! Thanks good catch. Done.